server/neptune/temporalworker/server.go

```diff
@@ -60,14 +43,15 @@ type Server struct {
 	StatsCloser     io.Closer
 	TemporalClient  client.Client
 
-	DeployActivities    workflows.DeployActivities
-	TerraformActivities workflows.TerraformActivities
+	DeployActivities    *workflows.DeployActivities
+	TerraformActivities *workflows.TerraformActivities
+	GithubActivities    *workflows.GithubActivities
```

Comment:
I'm a little confused about why GithubActivities were split into ints own field here. What was the motiviation?

Reply:
As discussed during standup, I separated out github activities to avoid registering the activity twice since both the deploy and terraform workflow uses it.